sed/calibrator/energy.py

```diff
@@ -2082,8 +2103,7 @@ def fit_energy_calibration(
     vals: list[float] | np.ndarray,
```

Comment:
the vals variable and other such can have more descriptive names so one doesn't need to go through the docstrings to understand

Reply:
I would not add more changes to the existing code now. I can open an issue. As for the first point, if the bias voltage is not available, you still get a "correct" energy axis, but offset by the bias voltage applied. So I would not consider this a critical failure, but something to notify about.